.bettercodehub.yml

```diff
@@ -0,0 +1,21 @@
+component_depth: 1
+default_excludes: true
+languages:
+- name: java
+  production:
+    include:
+      - server/src/main/.*\.java
+  test:
+    include:
+      - server/src/test/.*\.java
+- name: typescript
+  production:
+    include:
+      - client/src/app/.*\.ts
+      - client/src/testing/.*\.ts
```

Comment:
Shouldn't this be in the test section?
Also I suggest adding `client/src/environments/.*.ts`